src/hooks/useAppConfig.tsx

```diff
@@ -118,15 +118,27 @@ export interface AppConfig {
   };
 }
 
-export const useAppConfig = () => {
+export const useAppConfigList = () => {
   const { activeProject } = useActiveProject();
 
   const query = useRestQuery(
-    'GET /v1/life-research/projects/:projectId/app-config',
+    'GET /v1/life-research/projects/:projectId/app-configs/list',
```

Comment:
Somewhere we will need to make this smarter, probably at the API level. Currently this list will contain all configs that exist for the project but an App user should likely only be able to change between configs that they belong to (cohort or user list). This is probably okay until we identify how to manage that logic. I imagine it could end up being different per client.

Reply:
Total oversight on my part, I'll work on updating API side